.github/pull_request_template.md

```diff
@@ -1,6 +1,8 @@
 I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://raw.githubusercontent.com/risingwavelabs/risingwave/17af8a747593ebdbfa826691daf75bdab7d14fa0/.github/contributor-license-agreement.txt).
 
-Highest Release Version: 2.3
+[Release Timeline](https://github.com/risingwavelabs/rw-commits-history/blob/main/release_timeline.md)
+[Currently Supported Versions](https://docs.risingwave.com/changelog/release-support-policy#support-end-dates-for-recent-releases)
```

Comment:
still confused why PR author needs to be care about these.

Should this be included somewhere in the checklist?

Reply:
I’m fine with moving these two lines to the checklist section:

> My PR contains critical fixes that are necessary to be merged into the latest release.

Also, instead of saying "latest release", we should update it to "the latest several releases", as defined by our release policy.

I hope these two links can help remind the PR author which branches require cherry-picking, and what release stage we’re currently in.

Reply:
sounds good